Modules/Keychain/Sources/Secured.swift

```diff
@@ -0,0 +1,173 @@
+import Foundation
+
+///
+/// `Secured` is a property wrapper that can be applied to any codable property to store it in the keychain
+///
+/// Example:
+/// ```swift
+/// @Secured(key: "keychain.value_key")
+/// var value: Bool
+/// ```
+@propertyWrapper
+public struct Secured<Value: Codable> {
+    private enum KeychainError: Error, LocalizedError {
+        case itemNotFound
+        case accessError(status: String)
+        case unexpectedData
+        case decodingError(error: Error)
+        case encodingError(error: Error)
+        case saveItemToKeychain(status: String)
+        case deleteItem
+
+        var errorDescription: String? {
+            switch self {
+            case .itemNotFound: return "The requests item could not be found in the keychain"
+            case .accessError: return "The keychain item could not be accessed"
+            case .unexpectedData: return "The retrieved keychain data type is unexpected"
+            case .decodingError: return "The retrieved keychain data could not be decoded"
+            case .encodingError: return "The item could not be encoded"
+            case .saveItemToKeychain: return "The item could not be saved to the keychain"
+            case .deleteItem: return "The item could not be removed from the keychain"
+            }
+        }
+
+        var failureReason: String? {
+            switch self {
+            case let .accessError(status: error): return error
+            case let .decodingError(error: error): return error.localizedDescription
+            case let .encodingError(error: error): return error.localizedDescription
+            case let .saveItemToKeychain(status: error): return error
+            default: return nil
+            }
+        }
+
+        var recoverySuggestion: String? {
+            switch self {
+            case .itemNotFound: return "Please make sure that the item exists in the keychain before you try to access it"
+            case .deleteItem: return "Please make sure that the item exists in the keychain before you try to delete it"
+            default: return nil
+            }
+        }
+
+        func description(for key: String) -> String {
+            var description: String = .init(format: "An keychain error associated with the key: %@ occurred",key)
```

Comment:
Please adjust the description to "A keychain error associated with the key: %@key occurred". Furthermore add a blank space between the comma and "key"